infra/notifications/src/browserPush.js

```diff
@@ -0,0 +1,83 @@
+const Sequelize = require('sequelize')
+const PushSubscription = require('./models').PushSubscription
+const { getNotificationMessage } = require('./notification')
+const webpush = require('web-push')
+
+//
+// Browser push subscripttions
+//
+async function browserPush(
+  eventName,
+  party,
+  buyerAddress,
+  sellerAddress,
+  offer
+) {
+  console.log('🖥 Browser Push')
+  if (!eventName) throw 'eventName not defined'
```

Comment:
MobilePush, browserPush and emailPush all check for the presence of these properties. Might make sense to move it out into one place. Perhaps at the bottom of `src/app.js` where they are all called

Reply:
I think these guards are useful: E.g. in development I had a typo and was accidentally passing in a null value as one of the addresses. This caused things to fail in subtle ways that were a pain to debug. Better for these functions to protect themselves against being calling with invalid params.